input_parser.star

```diff
@@ -807,15 +807,6 @@ def args_sanity_check(plan, deployment_stages, args, user_args, op_stack_args):
         if deployment_stages.get("deploy_optimism_rollup", False):
             fail("Gas token is not supported when deploying OP Rollup.")
 
-        # Validate the gas token address.
-        gas_token_address = args.get("gas_token_address", "")
-        if gas_token_address == constants.ZERO_ADDRESS or gas_token_address == "":
-            fail(
-                "Gas token is enabled, but the provided gas token address is either empty or set to the zero address: '{}'.".format(
-                    gas_token_address
-                )
-            )
-
```

Comment:
why did you remove this check?

Reply:
because if i am passing `gas_token_enabled` as true, then this check is failing. i need to pick up the `gas_token_address` dynamically which i am doing it here https://github.com/0xPolygon/kurtosis-cdk/pull/547/files#diff-c6404e3bb415433745b3dad96f7be3df9fa4ed23eae7a5431e409e7f142395f2R105